app/controllers/countries_controller.rb

```diff
@@ -0,0 +1,7 @@
+class CountriesController < ApplicationController
```

Comment:
No need for a CountriesController. As they will be used on a downloads controller.